src/main/java/org/openrewrite/java/testing/jmockit/ExpectationsBlockRewriter.java

```diff
@@ -22,6 +22,7 @@
 import org.openrewrite.java.JavaVisitor;
 import org.openrewrite.java.tree.*;
 
+import javax.annotation.Nullable;
```

Comment:
I would avoid `javax.annotation` due to the Java EE / Jakarta EE debacle. No other recipes in this repo use `javax.annotation.Nullable`, I'd recommend we keep it that way.

Reply:
I don't know about that debacle - more details? Do you you also prefer not using null as a valid argument? Perhaps it's better as I remember having a discussion around this with a reviewer l last year and ended up not using null, I forgot about it!

Reply:
It's a big pain related to the ownership of the Java EE transferring from Oracle to the Eclipse Foundation. Oracle apparently required them to change the naming to differentiate between Oracle and non-Oracle owned software.

See the [original Oracle post](https://blogs.oracle.com/javamagazine/post/transition-from-java-ee-to-jakarta-ee). Also see how the Spring community made the big decision to [move to Jakarta EE](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide#jakarta-ee) in Spring Boot 3.

In practice, this means that the `javax.annotation` package is no longer maintained and has been effectively replaced by `jakarta.annotation` (along with the dependencies themselves). It's also apparently not a good idea to mix the two together. So either you should be using all Java EE (effectively tech debt), or use all Jakarta EE in your project.

In my software I avoid Java EE unless the project I'm working in already has them. `@Nullable` is nice but it's not functionally necessary.

Reply:
Thanks for pointing this out @tinder-dthomson ; I've now codified this check in https://github.com/openrewrite/rewrite-recommendations/commit/3d724859e0137bc4ea3d7be845e631f5ed9eddfd, such that it will run on all PR going forward.